css/_variables.scss

```diff
@@ -89,7 +89,7 @@ $sidebarWidth: 200px;
  */
 $tooltipsZ: 901;
 $toolbarZ: 900;
-$overlayZ: 800;
```

Comment:
Also I think it looks very strange to have a toolbar with buttons that you cannot use. If we are doing this on purpose probably we should at least not showing the toolbar in these cases and currently the toolbar is displayed even when the overlay is displayed.